Makefile.am

```diff
@@ -357,6 +359,9 @@ yamllint:
 		echo $${YAML}; \
 		$(PYTHON) -c "import yaml; f = open('$${YAML}'); yaml.safe_load(f); f.close()" || { echo Your YAML file: $${YAML} has a wrong syntax or this is a Jinja template. In the latter clause, consider to add your YAML file to the YAML_TEMPLATE_FILES list in Makefile.am.; exit 1; } \
 	done; \
+	echo -e "\nCheck PRCI definitions"; \
+	echo "-----------"; \
+	$(PYTHON) $(top_srcdir)/ipatests/prci_definitions/prci_checker.py -d $(top_srcdir)/ipatests/prci_definitions -s $(top_srcdir)/ipatests/prci_definitions/prci_jobs_spec.yaml; \
```

Comment:
@dav-pascual.
The result of `prci_checker.py` is hidden with the following `echo`. Bash's `exit on error` mode is not enabled for this make's target.

```console
[builder@localhost freeipa]$ make yamllint && echo Xpassed
...
Check PRCI definitions
-----------
BEGINNING PRCI JOB DEFINITIONS CHECKS
[File] nightly_latest_testing [Jobs] 145
[File] nightly_latest [Jobs] 145
[File] nightly_latest_testing_selinux [Jobs] 145
[File] nightly_latest_selinux [Jobs] 145
[File] nightly_previous [Jobs] 145
[File] nightly_latest_pki [Jobs] 72
[File] gating [Jobs] 24
ERROR: In job 'fedora-latest/test_installation_TestInstallMaster':
  'class' field should be defined with one of the supported: ['RunPytest', 'RunPytest2', 'RunPytest3', 'RunWebuiTests', 'RunADTests']
[File] nightly_rawhide [Jobs] 145
[File] nightly_latest_sssd [Jobs] 16
[File] nightly_latest_389ds [Jobs] 45
CHECKS FINISHED WITH ERRORS
-----------
Xpassed
```